elements/pfe-sass/mixins/_copy-mixins.scss

```diff
@@ -208,25 +208,25 @@
             --pf-c--FontSize: #{pfe-local(FontSize)};
         }
         @if $use-local {
+            font-family: pfe-local(LineHeight, $font-family, $region)#{$rules};
             font-size: pfe-local(FontSize, $font-size, $region)#{$rules};
             @if $base { 
-                font-family: pfe-local(LineHeight, $font-family, $region)#{$rules};
                 line-height: pfe-local(LineHeight, $line-height, $region)#{$rules};
                 font-weight: pfe-local(FontWeight, $font-weight, $region)#{$rules};
-            }
-            &:not(:last-child) {
-                margin-bottom: pfe-local(MarginBottom, $margin-bottom, $region)#{$rules};
+                &:not(:last-child) {
+                    margin-bottom: pfe-local(MarginBottom, $margin-bottom, $region)#{$rules};
+                }
             }
         }
         @else {
+            font-family: $font-family#{$rules};
             font-size:   $font-size#{$rules};
             @if $base {  // these styles are ordinarily inherited from the body tag, but some projects may need extra specificity
-                font-family: $font-family#{$rules};
                 line-height: $line-height#{$rules};
                 font-weight: $font-weight#{$rules};
-            }
-            &:not(:last-child):not(:empty) {
-                margin-bottom: $margin-bottom#{$rules}; //16px
+                &:not(:last-child):not(:empty) {
```

Comment:
should the pfe-typography mixin allow you to opt out of margins separately from the other font styles? so if i want font-family, font-size, line-height, and font-weight, I can get those things but avoid the margins

Reply:
The alternative is you must skip the mixin and call things one by one:
    font-family: pfe-var(font-family--heading);
    font-size: pfe-var(FontSize--xl);
    line-height: pfe-var(line-height);
    font-weight: pfe-var(font-weight--normal);

Reply:
I thought the goal of base was to provide attributes that are generally inherited like font-family and line-height?

Comment from line 224 supports this:
`// these styles are ordinarily inherited from the body tag, but some projects may need extra specificity`

Reply:
Since margin is not inherited, I'm not sure it makes sense to invoke it with base

Reply:
++ Potter said the same when we were reviewing , I'm going to break that one out into a new argument